sale_report_purchase_price/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Adds `purchase_price` field to sale report.
```

Comment:
Add here an explanation of what this field means (which is not the real purchase price).